DynamicActiveQuery.php

```diff
@@ -216,7 +216,7 @@ public function createCommand($db = null)
             % (`?) \(! \s*?
                 ( [a-z_\x7f-\xff][a-z0-9_\x7f-\xff]* (?: \. [^.|\s]+)* )
                 (?:  \| (binary (?:\(\d+\))? | char (?:\(\d+\))? | time (?:\(\d+\))? | datetime (?:\(\d+\))? | date
-                        | decimal (?:\(\d\d?(?:,\d\d?)?\))?  | double (?:\(\d\d?,\d\d?\))?
+                        | decimal (?:\(\d\d?(?:,\d\d?)?\))?  | double (?:\(\d\d?,\d\d?\))? | numeric? | text ? |
                         | int(eger)? | (?:un)? signed (?:\s+int(eger)?)?)  )?
             \s*? !\) \1 %ix
 REGEXP;
```

Comment:
if you want to use the same regex for pgsql and maria then we must test that all acceptable patterns behave properly. e.g. do things like `decimal`, `datetime` and `double` work in pgsql? do `numeric` and `text` work in maria?

if not then the exception should make clear what the problem is.